ru/mail/chrome/communicator/wallet/SignonViewer.dtd

```diff
@@ -1,14 +1,14 @@
-<!ENTITY      windowtitle.label               "Менеджер паролей">
+﻿<!ENTITY      windowtitle.label               "Менеджер паролей">
 <!ENTITY      windowalttitle.label            "Менеджер форм: Сайты">
```

Comment:
@yshchebet @Kostrikov  This is an opportunity to get things correct.  If there is a better translation, we should use it.  

We're going through the trouble of doing this, so let's do it right.